microsoft-edge/devtools-guide-chromium/overview.md

```diff
@@ -129,6 +129,8 @@ To have DevTools automatically open whenever you open a new tab in the browser:
    microsoft-edge-dev --auto-open-devtools-for-tabs
    ```
    
+> [!IMPORTANT]
+> There must be no running Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable Startup Boost in `edge://settings/system`. More information about Startup Boost can be found here: [Get help with startup boose - Microsoft Support](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).
```

Comment:
```suggestion

###### Disabling startup boost

> [!IMPORTANT]
> There must be no running Microsoft Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable **Startup boost** in `edge://settings/system`. For more information, see [Get help with startup boost](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).
```
Rendered view (GitHub version):
https://github.com/ryanborMSFT/edge-developer/blob/patch-1/microsoft-edge/devtools-guide-chromium/overview.md#have-devtools-automatically-open-when-you-open-a-new-tab

Given the present style sheet, we're moving toward h2/h4/h6.  The containing heading is h3.  h6 should produce a suitable size at LMC (different than GitHub's rendering of heading sizes).

A heading is good to provide a skimmable label for an Alert eg an Important note.

Reply:
sent chat